changelog/3896.bugfix.rst

```diff
@@ -0,0 +1 @@
+Better handle rewrite lines wider than the terminal.  EG: xdist with 16+ cores and 1k+ testcases will no longer scroll manically while gathering testcases.
```

Comment:
s/Better handle rewrite/Improved handling of rewriting/

Reply:
Sorry, thought it should be past tense, but apparently it is not?!
The README does not really make it clear, but uses present tense itself (https://github.com/blueyed/pytest/blob/coverage/changelog/README.rst).
Then `s/…/Improve handling of rewriting/` still maybe.